tests/TestHelpers/GraphHelper.php

```diff
@@ -35,23 +35,10 @@ public static function getUUIDv4Regex(): string {
 	}
 
 	/**
-	 * @param string $id
-	 *
-	 * @return bool
-	 */
-	public static function isUUIDv4(string $id): bool {
-		$regex = "/^" . self::getUUIDv4Regex() . "$/i";
-		return (bool)preg_match($regex, $id);
-	}
-
-	/**
-	 * @param string $spaceId
-	 *
-	 * @return bool
+	 * @return string
 	 */
-	public static function isSpaceId(string $spaceId): bool {
-		$regex = "/^" . self::getUUIDv4Regex() . '\\$' . self::getUUIDv4Regex() . "$/i";
-		return (bool)preg_match($regex, $spaceId);
+	public static function getSpaceIdRegex(): string {
+		return '[0-9A-Fa-f]{8}-[0-9A-Fa-f]{4}-4[0-9A-Fa-f]{3}-[89ABab][0-9A-Fa-f]{3}-[0-9A-Fa-f]{12}\\\$[0-9A-Fa-f]{8}-[0-9A-Fa-f]{4}-4[0-9A-Fa-f]{3}-[89ABab][0-9A-Fa-f]{3}-[0-9A-Fa-f]{12}';
```

Comment:
can be built using `getUUIDv4Regex()`
```suggestion
		return getUUIDv4Regex() . '\\\$' . getUUIDv4Regex();
```